### ISSUE TITLE
Article title hover color

### ISSUE THREAD

**Issue by skourismanolis** [kinimatografiko-pofpa/bobina-web#8]
Hovering on an article should change it's title's color to primary red.

**Comment by mgiannopoulos24** [comment]
Hey there. I would like to work on this issue. Could you assign it to me if it is still incomplete?

**Comment by skourismanolis** [comment]
I don't think you should wait for me to assign stuff before doing any work, just mention it in a comment so others won't take it.

Thanks for the interest in the project! Looking forward to your contributions :3
